examples/recipes/qwen_vl/finetune_qwen_vl.py

```
@@ -103,9 +103,9 @@
     create_omegaconf_dict_config,
     parse_hydra_overrides,
 )
-from megatron.bridge.training.vlm_step import forward_step
+from megatron.bridge.training.qwen3vl_step import forward_step
 from megatron.bridge.utils.common_utils import get_rank_safe
-
+from functools import partial
```

Flake8 (7.3.0)

[error] 'functools.partial' imported but unused

(F401)